notebooks/Notebook2.md

```diff
@@ -9,6 +9,11 @@ Did it help you make a design decision? Was it helpful in addressing the most pr
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+Olivia mentioned that the syntax for splitting into two sections seemed a bit too verbose.
+I agree with this, but am at a slight loss in how to make it more concise. Currently, there 
+are 3 parts to it: First is the parts definition (where you say how many parts there will be and
+define their names. For example, A&B). Then, two more parts where each represents the different part (one would represent A, the other B). I'm not sure how I could make it more concise.
```

Comment:
Would it be possible to reduce the number of backticks or dashes that signify these sections - maybe just one instead of three or four? I think that would go a long way towards making these sections more concise!